### PR TITLE
Create translation-of-pancagatidipani_hazlewood-a.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Translation of Pancagatidipani"
+authors:
+  - hazlewood-a
+external_url: "https://ia904504.us.archive.org/32/items/jpts-xi-1987/Pancagatidipani_%20Illumination%20of%20the%20Five%20Realms%20of%20Existence%20-%20Ann%20Appleby%20Hazlewood_text.pdf"
```
Comment (buddhist-uni) [inline _content/articles/translation-of-pancagatidipani_hazlewood-a.md:5]: Please use the permalink, not the ephemeral URL:

https://archive.org/download/jpts-xi-1987/Pancagatidipani_%20Illumination%20of%20the%20Five%20Realms%20of%20Existence%20-%20Ann%20Appleby%20Hazlewood_text.pdf

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Translation of Pancagatidipani"
+authors:
+  - hazlewood-a
```
Comment (buddhist-uni) [inline _content/articles/translation-of-pancagatidipani_hazlewood-a.md:4]: Hazlewood doesn't have a `_authors` entry does she?

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/10jzF4gZhLTwEwvbB2MHWyI1FHe-9DXDl/view?usp=sharing"
+course: cosmology
+tags:
+  - karma
+  - dana
+  - sila
+year: 1987
+journal: jpts
+volume: 9
```
Comment (buddhist-uni) [inline _content/articles/translation-of-pancagatidipani_hazlewood-a.md:15]: Isn't it volume 11?

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+authors:
+  - hazlewood-a
+external_url: "https://ia904504.us.archive.org/32/items/jpts-xi-1987/Pancagatidipani_%20Illumination%20of%20the%20Five%20Realms%20of%20Existence%20-%20Ann%20Appleby%20Hazlewood_text.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/10jzF4gZhLTwEwvbB2MHWyI1FHe-9DXDl/view?usp=sharing"
+course: cosmology
+tags:
+  - karma
+  - dana
+  - sila
```
Comment (buddhist-uni) [inline _content/articles/translation-of-pancagatidipani_hazlewood-a.md:12]: `sila` isn't a tag

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+title: "Translation of Pancagatidipani"
+authors:
+  - hazlewood-a
+external_url: "https://ia904504.us.archive.org/32/items/jpts-xi-1987/Pancagatidipani_%20Illumination%20of%20the%20Five%20Realms%20of%20Existence%20-%20Ann%20Appleby%20Hazlewood_text.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/10jzF4gZhLTwEwvbB2MHWyI1FHe-9DXDl/view?usp=sharing"
+course: cosmology
+tags:
+  - karma
+  - dana
```
Comment (buddhist-uni) [inline _content/articles/translation-of-pancagatidipani_hazlewood-a.md:11]: Not really about dana?

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Translation of Pancagatidipani"
+authors:
+  - hazlewood-a
+external_url: "https://ia904504.us.archive.org/32/items/jpts-xi-1987/Pancagatidipani_%20Illumination%20of%20the%20Five%20Realms%20of%20Existence%20-%20Ann%20Appleby%20Hazlewood_text.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/10jzF4gZhLTwEwvbB2MHWyI1FHe-9DXDl/view?usp=sharing"
+course: cosmology
+tags:
+  - karma
```
Comment (buddhist-uni) [inline _content/articles/translation-of-pancagatidipani_hazlewood-a.md:10]: let's go for `roots` instead, as this is an excellent example of the Buddhist textual tradition

**Review by buddhist-uni — COMMENTED** [review]
No need for "translation-of-" in the filename 😊

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md (hunk 4 above)
Comment (JJM317) [reply _content/articles/translation-of-pancagatidipani_hazlewood-a.md:12]: This was my suggestion that it should be. I can take it out though.

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md (hunk 5 above)
Comment (JJM317) [reply _content/articles/translation-of-pancagatidipani_hazlewood-a.md:11]: Not about dana, but peripherally it does speak about giving and the rebirths it leads to. I can take it out though.

File: _content/articles/translation-of-pancagatidipani_hazlewood-a.md (hunk 2 above)
Comment (JJM317) [reply _content/articles/translation-of-pancagatidipani_hazlewood-a.md]: I saw how you marked Hazelwood's name in another entry and thought it was a slug from the look of it. I fixed it.

Comment (buddhist-uni) [reply _content/articles/translation-of-pancagatidipani_hazlewood-a.md]: Oh, you mean the filename?  Yeah, I can be a bit lazy and abbreviate in the filename sometimes. I think for that one in particular, I was anticipating eventually adding an entry for her in the future. No worries 😊

**Pull request merged**